### PR TITLE
apps/examples/err_report_demo: Fix error reporting demo application

### PR DESCRIPTION
**Pull request by dr-venkman**
1. Fix TASH command name in Makefile
2. Fix build error due to improper declaration of WiFi Manager call back for sta_disconnected event.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [29996bcec4cad83c0936a0e89f312338e02de0dc] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [29996bcec4cad83c0936a0e89f312338e02de0dc] - Code Rule Check (C++) OK.

File: apps/examples/err_report_demo/err_report_main.c [hunk 1]
```diff
@@ -84,7 +84,7 @@ uint8_t g_err_nsuccess;
  * WiFi Manager callback prototypes
  */
 static void prv_sta_connected(wifi_manager_result_e);
-static void prv_sta_disconnected(void);
+static void prv_sta_disconnected(wifi_manager_disconnect_e);
```
Comment (sunghan-chang) [inline apps/examples/err_report_demo/err_report_main.c]: You should fill the argument name. That's our coding rules.

Comment (dr-venkman) [reply apps/examples/err_report_demo/err_report_main.c]: Modified as suggested. Thank you.

**Comment by seinfra** [comment]
Target : [a248464fb76e5947279246b08b7482903839bb7e] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a248464fb76e5947279246b08b7482903839bb7e] - Code Rule Check OK.

**Pull request merged**